src/net_processing.cpp

```diff
@@ -3855,15 +3861,20 @@ bool PeerLogicValidation::ProcessMessages(CNode* pfrom, std::atomic<bool>& inter
     //
     bool fMoreWork = false;
 
-    if (!pfrom->vRecvGetData.empty())
-        ProcessGetData(*pfrom, chainparams, m_connman, m_mempool, interruptMsgProc);
+    {
+        LOCK(pfrom->cs_vRecv);
+        if (!pfrom->vRecvGetData.empty())
```

Comment:
style: consider adding braces to if blocks when touching code, to conform with the current style guide.

Reply:
done! did not do so to adjacent code i wasn't touching.